### DISCUSSION TITLE
Hide cookie banner by default, so accept required and show the cookie icon

### DISCUSSION THREAD

**Discussion by notflip** [dargmuesli/nuxt-cookie-control#169] [Q&A] [answered]
Is it possible to just accept the required cookies by default, and have the icon with the cookie show on the bottom right, without the big banner? Thanks!

**Comment by dargmuesli** [comment, marked as answer]
Of course, this module offers the possibility to select cookies. You could configure cookies to be enabled by default in some initialization function, but that would defeat the legal purposes of this module.
